Update metadata-syntax-for-github-actions.md

mambax opened the pull request:
I tried to use another name than Dockerfile, the build fails saying:

`docker: invalid reference format: repository name must be lowercase.`

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

mambax committed:
Update metadata-syntax-for-github-actions.md

I tried to use another name than Dockerfile, the build fails saying:

`docker: invalid reference format: repository name must be lowercase.`

janiceilene commented:
@mambax Thanks so much for opening a PR! I'll get this triaged for review ✨

janiceilene commented:
@mambax It looks like our new action inadvertently closed this 😢 I'm so sorry about that! We're merging a fix right now.

mambax commented:
@janiceilene I thought its a bit weird. Should I restore my branch?

mambax commented:
/reopen

janiceilene commented:
Thank you so much for restoring your branch! I'll get this back up for review 💖

hubwriter commented on content/actions/creating-actions/metadata-syntax-for-github-actions.md:
```diff
@@ -297,7 +297,7 @@ runs:
 #### `runs.image`
 
-**Required** The Docker image to use as the container to run the action. The value can be the Docker base image name, a local `Dockerfile` in your repository, or a public image in Docker Hub or another registry. To reference a `Dockerfile` local to your repository, use a path relative to your action metadata file. The `docker` application will execute this file.
+**Required** The Docker image to use as the container to run the action. The value can be the Docker base image name, a local `Dockerfile` in your repository, or a public image in Docker Hub or another registry. To reference a `Dockerfile` (the filename must be `Dockerfile`) local to your repository, use a path relative to your action metadata file. The `docker` application will execute this file ...
```
I think the sentence might be simpler to parse without the parentheses. Maybe change this to:
```suggestion
**Required** The Docker image to use as the container to run the action. The value can be the Docker base image name, a local `Dockerfile` in your repository, or a public image in Docker Hub or another registry. To reference a `Dockerfile` local to your repository, the file must be named `Dockerfile` and you must use a path relative to your action metadata file. The `docker` application will execute this file.
```

hubwriter reviewed (changes requested):
@mambax - Many thanks for pushing this PR. It'll be good to clarify this point that isn't currently clear. We appreciate you helping us improve the docs! Thanks again. 👍

mambax committed:
Update content/actions/creating-actions/metadata-syntax-for-github-actions.md

Thanks, I struggled to find the correct words and not change the text too much

Co-authored-by: hubwriter <hubwriter@github.com>

hubwriter reviewed (approved):
@mambax - great, thanks again, I'll get this merged so that this change goes live. 🎉

hubwriter committed:
Merge branch 'main' into patch-1

hubwriter committed:
Merge branch 'main' into patch-1

hubwriter committed:
Merge branch 'main' into patch-1
